core/consensus/validation/babe_block_validator.hpp

```diff
@@ -86,9 +91,42 @@ namespace kagome::consensus {
      * @return true if vrf is valid, false otherwise
      */
     bool verifyVRF(const BabeBlockHeader &babe_header,
+                   const EpochIndex epoch_index,
                    const primitives::BabeSessionKey &public_key,
                    const Threshold &threshold,
-                   const Randomness &randomness) const;
+                   const Randomness &randomness,
+                   const bool checkThreshold) const;
+
+    inline primitives::Transcript &makeTranscript(
+        primitives::Transcript &transcript_,
```

Comment:
It's somewhat confusing that it takes a transcript as a parameter and is called makeTranscript. Maybe something like prepareTranscript, fillTranscript or bakeTranscript will be a bit more clear.

Reply:
```prepareTranscript``` sounds good.